From: KevinDuan <duankaiwen@kylinos.cn>
Date: Thu, 20 Aug 2026 03:35:12 +0000
Subject: Translated using Weblate (Portuguese)

Currently translated at 100.0% (11 of 11 strings)

Translation: openkylin-huanghe/ukui-task-manager
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-task-manager/pt/
---
 .../translations/libukui-task-manager_pt.ts        | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/widgets/ukui-task-manager/translations/libukui-task-manager_pt.ts b/widgets/ukui-task-manager/translations/libukui-task-manager_pt.ts
index 760d743..801dabc 100644
--- a/widgets/ukui-task-manager/translations/libukui-task-manager_pt.ts
+++ b/widgets/ukui-task-manager/translations/libukui-task-manager_pt.ts
@@ -5,56 +5,56 @@
     <name>TaskManager::TaskManagerItem</name>
     <message>
         <source>Add launcher to panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Fixar na barra de tarefas</translation>
     </message>
     <message>
         <source>Remove launcher from panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Desafixar da barra de tarefas</translation>
     </message>
     <message>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Fechar</translation>
     </message>
 </context>
 <context>
     <name>TaskManager::UkuiTaskManager</name>
     <message>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Fechar</translation>
     </message>
     <message>
         <source>Restore</source>
-        <translation type="unfinished"></translation>
+        <translation>Restaurar</translation>
     </message>
     <message>
         <source>Maximize</source>
-        <translation type="unfinished"></translation>
+        <translation>Maximizar</translation>
     </message>
     <message>
         <source>Minimize</source>
-        <translation type="unfinished"></translation>
+        <translation>Minimizar</translation>
     </message>
     <message>
         <source>Keep above</source>
-        <translation type="unfinished"></translation>
+        <translation>Manter no topo</translation>
     </message>
     <message>
         <source>Unset keep above</source>
-        <translation type="unfinished"></translation>
+        <translation>Cancelar manter no topo</translation>
     </message>
 </context>
 <context>
     <name>ThumbnailContainer</name>
     <message>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Fechar</translation>
     </message>
 </context>
 <context>
     <name>ThumbnailWindow</name>
     <message>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Fechar</translation>
     </message>
 </context>
 </TS>
